docs/markdown/Reference-tables.md

```diff
@@ -11,6 +11,7 @@ These are return values of the `get_id` (Compiler family) and
 | armclang  | ARMCLANG compiler                |                 |
 | ccomp     | The CompCert formally-verified C compiler |        |
 | ccrx      | Renesas RX Family C/C++ compiler |                 |
+| iccrl78   | The IAR C compiler (RL78 Family) |                 |
```

Comment:
Not knowing much about IAR internals how different are the various compilers? It seems like the compiler name should be `iar` and then get the platform info from `cpu` and the like.

Reply:
How much do they differ from each other? Do they take all the same compiler arguments etc?